test/Pester/PlatyPs.Tests.ps1

```diff
@@ -517,6 +517,113 @@ Get-Alpha [-WhatIf] [[-CCC] <String>] [[-ddd] <Int32>] [<CommonParameters>]
             normalizeEnds(Get-Content $files | Where-Object {$_.StartsWith('Get-Alpha')} | Out-String) | Should Be $expectedSyntax
         }
     }
+
+    Context 'DontShow parameter' {
```

Comment:
Let's just stick with something basic, no need to solve all the corner cases in a single go. I think in the real help we will have hardly any cases where one parameter is NoShow in one set and not NoShow in another.